content/Developers_Handbook/Kafka/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Kafka
+weight: 11
+---
+
+## Read messages from Kafka
+
+### Number of partitions and their message count 
+
+Substitute `dataset-events` with the desired topic.
```

Comment:
Hvor mange andre `topics` er det? Kanskje nevne to-tre eksempler til, og/eller hvor en finner oversikt over topics.

Reply:
La til cmd for å liste opp alle topics. Beste oversikten får man av å se hva som finnes, dokumentasjon kan bli utdatert.